themes/bootstrap3/templates/record/cover.phtml

```diff
@@ -1,8 +1,8 @@
 <?php /* Display thumbnail if appropriate: */ ?>
 <?php if ($cover): ?>
   <?php if ($this->link): ?><a href="<?=$this->escapeHtmlAttr($this->link)?>" data-lightbox-image><?php endif; ?>
-  <img alt="<?=$this->transEsc('Cover Image')?>" <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>class="recordcover" src="<?=$this->escapeHtmlAttr($cover); ?>"/>
+  <img src="<?=$this->escapeHtmlAttr($cover); ?>" <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>class="recordcover" alt="<?=trim($this->driver->tryMethod('getTitle') ?? $this->translate("Cover Image")); ?>" />
   <?php if ($this->link): ?></a><?php endif; ?>
 <?php else: ?>
-  <img src="<?=$this->url('cover-unavailable')?>" <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>class="recordcover" alt="<?=$this->transEsc('No Cover Image')?>"/>
+  <img src="<?=$this->url('cover-unavailable')?>" <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>class="recordcover" alt="<?=trim($this->driver->tryMethod('getTitle') ?? ""); ?>" />
```

Comment:
Is there any reason to have an alt on the "no cover" image? That's a purely visual element to make things line up neatly, but I don't think a screen reader user cares at all. (I think you could debate the value of having an alt on the cover image too, but since that is sometimes wrapped in a link tag, that justifies it).

Reply:
I agree with your assessment about the cover link (and don't have a strong opinion about a solution)... but my comment specifically referred to the "cover unavailable" image, which I'm pretty sure is never linked, and thus should be safe to use `alt=""` on.

Reply:
...and I see that you figured that out already. :-)